lib/notion/getAllPageIds.js

```
@@ -1,14 +1,20 @@
-    pageIds = views[vId]?.blockIds
+    const view = getRecordValue(views[vId])
+    pageIds = view?.blockIds
+    if (!pageIds) {
+      pageIds = view?.collection_group_results?.blockIds
+    }
   } else {
     const pageSet = new Set()
     Object.values(views).forEach(view => {
-      view?.collection_group_results?.blockIds?.forEach(id => pageSet.add(id))
+      const viewValue = getRecordValue(view)
+      viewValue?.collection_group_results?.blockIds?.forEach(id => pageSet.add(id))
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)